gap/base/recognition.gi

```diff
@@ -854,6 +854,9 @@ InstallGlobalFunction( "SLPforNiceGens", function(ri)
   return s;
 end );
 
+# For user debugging purposes. Takes a string consisting of lower- or
+# upper-case "f" and "k". Returns the node one arrives at by descending through
+# the tree, going to the factor for each "f" and to the kernel for each "k".
```

Comment:
What happens if one tries a non existing path?

Reply:
Either ImageRecogNode and KernelRecogNode should error.  Since it's a debugging function that's good enough IMO. If they are worth it, We could move all of those debug functions into a separate file, and provide short documentations.

Reply:
NVM, that may be the only function that is only used for debugging.

Reply:
Ok, thanks! 👍